docs/data/charts/components/components.md

```diff
@@ -79,29 +81,25 @@ By using `invert`, the value associated with the current mouse coordinate `y` ca
 
 ### Series
 
-Series information is accessible through the `useSeries` hook for all series types, and `useXxxSeries` hook for a specific series type.
-These hooks return the order of the series and their configuration, including data points, color, among others.
+Series information is accessible through the `useSeries()` hook for all series types, and the `useXxxSeries()` hook for a specific series type.
+These hooks return the order of the series and their configuration, including data points, color, and more.
 
-You can leverage that information to create custom charts.
-For example, you can use `useLineSeries` to obtain the series of a line chart and display an indicator of the minimum and maximum values of each series:
+You can use that information to create custom charts.
+For example, you can use `useLineSeries()` to obtain the series of a Line Chart and display an indicator of the minimum and maximum values of each series:
 
 {{"demo": "SeriesDemo.js"}}
 
 ## HTML components
 
-With the introduction of the `ChartDataProvider` in v8, the chart data can be accessed from any component.
-This lets you create HTML components that interact with the charts data.
+Use the `<ChartDataProvider>` to access chart data from any component.
+This lets you create HTML components that interact with the data.
 
-In the next example, notice that `MyCustomLegend` component displays the series names and colors.
-This creates an html `table` element, which can be customized in any way.
+In the next example, notice that the `<MyCustomLegend>` component displays the series names and colors.
+This creates an HTML `<table>` element, which can be customized in any way.
 
 {{"demo": "HtmlLegend.js"}}
 
 :::warning
 Note that the HTML components are not part of the SVG hierarchy.
-Hence, they should be:
-
-- Outside the `<ChartsSurface />` component to avoid mixing HTML and SVG.
-- Inside the `<ChartDataProvider />` component to get access to the data.
-
+This means they must be outside of the `<ChartsSurface>` component to avoid mixing HTML and SVG, and inside of the `<ChartDataProvider>` component to get access to the data.
```

Comment:
```suggestion
This means they must be outside of the `<ChartsSurface />` component to avoid mixing HTML and SVG, and inside of the `<ChartDataProvider />` component to get access to the data.
```